usr/src/test/smartos-test/smartos-test.sh

```diff
@@ -300,6 +300,17 @@ EOF
     fi
 }
 
+#
+# *_test_check() functions exist because some illumos tests assume some
+# reality checks, test drivers, or other prerequisites, that we cannot just
+# extract from the tests-tar tarball.
+#
```

Comment:
It looks like this comment has been accidentally separated from the functions it is commenting.

Reply:
Will fix.  (The one function is a support one for the *_test_check() ones.)

Reply:
Fixed.